packages/slice-machine/tests/components/Desktop.spec.tsx

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
I don't the value of this test because you create a new App and you test it. So it' not link to the real context

Reply:
yeah adding a large context is on my list of things to do, but now it checks that if the version is out of date the notification shows up (which could break when refactoring)